ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/activityBlock/activityBlock.component.html

```diff
@@ -9,6 +9,7 @@
                                           [validationRequested]="validationRequested"
                                           [studyGuid]="studyGuid"
                                           [parentActivityInstanceGuid]="parentActivityInstanceGuid"
```

Comment:
Not one of your changes in this PR, but do we need parentActivityInstanceGuid Input param? I expect it would be part `instance`.

Reply:
You are right. It is not even used there.
Removed at be6072935285637198cec61156d4196b7de1ea66.